src/loader-components/_PropsTable.jsx

```diff
@@ -3,80 +3,105 @@ import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import { ThemeContext } from '@cerner/terra-application/lib/theme';
+import Table, {
+  Header,
+  HeaderCell,
+  Body,
+  Cell,
+} from 'terra-html-table';
 import styles from './PropsTable.module.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-   * The props table rows.
+   * Children for the component.
    */
-  rows: PropTypes.arrayOf(
-    PropTypes.shape({
-      name: PropTypes.string,
-      type: PropTypes.func,
-      required: PropTypes.bool,
-      defaultValue: PropTypes.string,
-      description: PropTypes.func,
-    }),
-  ),
+  children: PropTypes.node,
 };
 
-const PropsTable = ({ rows }) => {
+const requiredCellPropTypes = {
+  /**
+   * Is the cell required?
+   */
+  isRequired: PropTypes.bool,
+};
+
+const PropNameCell = ({ children }) => (
+  <Cell
+    className={cx('bold')}
+    key="PROP"
```

Comment:
I don't think these keys are necessary, but they don't hurt anything either if you like having them.

Reply:
I added them based on the html table guides. 🤷‍♂️

Reply:
The guide might be mapping over a data set and building an array of Cells, in which case the keys would be necessary.

Reply:
Removed